odoo-shippable/scripts/library.sh

```diff
@@ -58,3 +58,20 @@ createuser_custom(){
 psql_create_role(){
     su - postgres -c "psql -c  \"CREATE ROLE ${1} LOGIN PASSWORD '${2}' SUPERUSER INHERIT CREATEDB CREATEROLE;\""
 }
+
+service_postgres_without_sudo(){
+    USER="${1}"
+    VERSIONS=$(pg_lsclusters  | sed '1d' | awk '{print $1}' )
+    for version in $VERSIONS; do
+        pg_dropcluster --stop $version main
+    done
+    adduser ${USER} postgres
+    chown -R ${USER}:${USER} /var/run/postgresql
```

Comment:
Should be assigned for postgres group?
I mean, `chown -R ${USER}:postgres /var/run/postgresql`